play-json/shared/src/main/scala/play/api/libs/json/JsResult.scala

```diff
@@ -207,6 +207,12 @@ sealed trait JsResult[+A] { self =>
     def withFilter(q: A => Boolean) = new WithFilter(a => p(a) && q(a))
   }
 
+  /** If this result is successful than checks for presence for '''elem''', otherwise return '''false''' */
+  def contains[AA >: A](elem: AA): Boolean = asOpt.contains(elem)
```

Comment:
Rather move implementation in the subtypes, as for the other functions.

Reply:
Done.